ds_judgements_public_ui/sass/includes/_footer.scss

```diff
@@ -82,3 +82,16 @@
     font-size: $typography-md-text-size;
   }
 }
+
+.more-info-footer {
+  text-align: center;
```

Comment:
Rather than rely on the contents of this being display inline, can we style this with flex so the contents is always styled in the middle, i.e.

```
display: flex;
flex-direction: column;
justify-content: center;
```

Reply:
@jlhdxw the prefooter below with the button has the exact same so I did the same display inline

Reply:
@jlhdxw updated both pre-footer which previously had inline  and more-info-footer with flex now